legend-engine-xt-persistence-test-runner/src/test/resources/bitemporal-delta/batch_id_and_date_time_based/persistence_no_del_ind_user_specifies_from_and_thru.txt

```diff
@@ -113,7 +113,7 @@ Persistence test::TestPersistence
                  ExternalFormat
                  #{
                    contentType: 'application/json';
-                   data: '[{"ID":1, "AMOUNT":100, "SOURCE_FROM": "2022-01-01 00:00:00.0", "SOURCE_THROUGH": "2022-01-02 00:00:00.0", "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "BATCH_IN":1, "BATCH_OUT":999999999},{"ID":2, "AMOUNT":200, "SOURCE_FROM": "2022-01-01 00:00:00.0", "SOURCE_THROUGH": "2022-01-02 00:00:00.0", "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "BATCH_IN":1, "BATCH_OUT":999999999},{"ID":3, "AMOUNT":400, "SOURCE_FROM": "2022-01-01 00:00:00.0", "SOURCE_THROUGH": "2022-01-02 00:00:00.0", "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "BATCH_IN":1, "BATCH_OUT":999999999}]';
+                   data: '[{"ID":1, "AMOUNT":100, "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "BATCH_IN":1, "BATCH_OUT":999999999},{"ID":2, "AMOUNT":200, "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "BATCH_IN":1, "BATCH_OUT":999999999},{"ID":3, "AMOUNT":400, "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "FROM_Z": "2022-01-01 00:00:00.0", "THRU_Z": "2022-01-02 00:00:00.0", "BATCH_IN":1, "BATCH_OUT":999999999}]';
```

Comment:
Believe we can remove one set of "FROM_Z" and "THRU_Z" from each assert, for now looks like it's repeated twice for each row

Reply:
fixed in the latest commit!